File: src/source/Yaaf.Database/EmptyConfiguration.cs
// ----------------------------------------------------------------------------
// This file is subject to the terms and conditions defined in
// file 'LICENSE.txt', which is part of this source code package.
// ----------------------------------------------------------------------------
using System;
using System.Collections.Generic;
using System.Data.Entity;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Yaaf.Database {

	public class EmptyConfiguration : DbConfiguration {
		public EmptyConfiguration ()
		{
			//SetDefaultConnectionFactory (new LocalDbConnectionFactory ("v11.0"));
		}
	}
}

File: src/source/Yaaf.Database/EmptyConfiguration.fs
﻿// ----------------------------------------------------------------------------
// This file is subject to the terms and conditions defined in
// file 'LICENSE.txt', which is part of this source code package.
// ----------------------------------------------------------------------------
namespace Yaaf.Database
open System
open System.Collections.Generic
open System.Data.Entity
open System.Linq
open System.Text
open System.Threading.Tasks

type EmptyConfiguration() =
    inherit DbConfiguration()
    do ()

type MSSQLConfiguration<'T when 'T :> DbContext>() as x =
  inherit System.Data.Entity.Migrations.DbMigrationsConfiguration<'T>()

  do
    x.AutomaticMigrationsEnabled <- false

  override x.Seed(context) = ()
